spec/article_json/export/apple_news/elements/heading_spec.rb

```diff
@@ -0,0 +1,31 @@
+describe ArticleJSON::Export::AppleNews::Elements::Heading do
+  subject(:element) { described_class.new(source_element) }
+
+  let(:source_element) do
+    ArticleJSON::Elements::Heading.new(content: 'Foo Bar', level: level)
+  end
+
+  describe '#export' do
+    subject { element.export }
+
+    context 'when the heading level is 1' do
+      let(:level) { 1 }
+      let(:heading) { { role: 'heading1', text: 'Foo Bar' } }
+      it { should eq heading }
+    end
+
+    context 'when the heading level is 2' do
+      let(:level) { 2 }
+      let(:heading) { { role: 'heading2', text: 'Foo Bar' } }
+      it { should eq heading }
+    end
+
+    (3..6).each do |i|
```

Comment:
Why not use the loop to cover all levels?
```suggestion
    (1..6).each do |i|
```

Reply:
Good point